force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +121,15 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static Map<Id, Task> getTasksDueInNext7Days() {
+        List<Task> tasks = [SELECT Id, Subject, ActivityDate 
+                            FROM Task
+                            WHERE ActivityDate = NEXT_N_DAYS:7];
+        
+        Map<Id, Task> tasksMap = new Map<Id, Task>(tasks);
```

Comment:
Excellent use of the special map constructor. You can actually put your query directly between the new Map constructor and not even bother with storing to a List.